.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Issue: <!-- link the issue or issues this PR resolves here -->
+<!-- If your PR depends on changes from another pr link them here and describe why they are needed in your solution section. -->
+
+## Problem
+<!-- Describe the root cause of the issue you are resolving. This may include what behavior is observed and why it is not desirable. If this is a new feature describe why we need this feature and how it will be used. -->
+## Solution
+<!-- Describe what you changed to fix the issue. Relate your changes back to the original issue / feature and explain how this addresses the issue. -->
+
+## Testing
+<!-- Note: Confirm if the repro steps in the GitHub issue are valid, if not, please update the issue with accurate repro steps. -->
+
+## Engineering Testing
+### Manual Testing
+<!-- Describe what manual testing you did (if no testing was done, explain why). -->
+
+### Automated Testing
+<!--If you added/updated unit/integration/validation tests, describe what cases they cover and do not cover. -->
+
+## QA Testing Considerations
+<!-- Highlight areas or (additional) cases that QA should test w.r.t a fresh install as well as the upgrade scenarios -->
+
+### Regressions Considerations
+<!-- Dedicated section to specifically call out any areas that with higher chance of regressions caused by this change, include estimation of probability of regressions -->
```

Comment:
Can we add another area to specifically call out if this chart change needs a backport or forwardport to any other rancher/charts branch?

Reply:
Yes! Will add it.

Reply:
Done!